packages/web-components/src/design-system-provider/index.ts

```diff
@@ -76,12 +77,25 @@ export class FluentDesignSystemProvider extends DesignSystemProvider
     this.noPaintChanged();
   }
 
+  @designSystemProperty({
+    attribute: 'neutral-base-color',
+    cssCustomProperty: false,
+    default: DesignSystemDefaults.neutralBaseColor,
+  })
+  public neutralBaseColor: string;
+  protected neutralBaseColorChanged(oldValue: string, newValue: string): void {
+    this.neutralPalette = createColorPalette(parseColorHexRGB(newValue));
```

Comment:
parseColorHexRGB can return `null` so you might want to null check the result prior to providing it to `createColorPalette` unless you want this to throw

Reply:
What's our standard here? I thought parseColor was the function that threw if the format of the color is not supported, but it looks now like it just runs a regex. What would be expected behavior if someone passed this something other than a valid hex color? Seems maybe preferable to throw instead of default to grey or something.